From patches Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
To: Peter Maydell <peter.maydell@linaro.org>, Gerd Hoffmann <kraxel@redhat.com>,
 qemu-devel@nongnu.org
Cc: =?utf-8?b?IkFuZHJlYXMgRsOkcmJlciI=?= <andreas.faerber@web.de>,
 peter.maydell@linaro.org, Gerd Hoffmann <kraxel@redhat.com>,
 qemu-devel@nongnu.org, Anthony Liguori <aliguori@us.ibm.com>
Date: Thu, 18 Apr 2013 08:20:23 +0200
Message-id: 1366266023-28322-1-git-send-email-kraxel@redhat.com
Subject: [PATCH 1/1] console: unbreak cocoa

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 ui/cocoa.m |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/cocoa.m b/ui/cocoa.m
index 048cc97..14b784d 100644
--- a/ui/cocoa.m
+++ b/ui/cocoa.m
@@ -495,7 +495,7 @@ QemuCocoaView *cocoaView;
                 if (keycode == 58 || keycode == 69) { // emulate caps lock and num lock keydown and keyup
                     kbd_put_keycode(keycode);
                     kbd_put_keycode(keycode | 0x80);
-                } else if (is_graphic_console()) {
+                } else if (qemu_console_is_graphic(NULL)) {
                     if (keycode & 0x80)
                         kbd_put_keycode(0xe0);
                     if (modifiers_state[keycode] == 0) { // keydown
@@ -1006,7 +1006,7 @@ static void cocoa_refresh(DisplayChangeListener *dcl)
             [cocoaView handleEvent:event];
         }
     } while(event != nil);
-    vga_hw_update();
+    graphic_hw_update(NULL);
 }
 
 static void cocoa_cleanup(void)
-- 
1.7.9.7

